### DISCUSSION TITLE
What order are files loaded in?

### DISCUSSION THREAD

**Discussion by hacker-DOM** [LazyVim/LazyVim#725] [Q&A]
I'm still struggling to understand this, could any of the contributors pls confirm this:

- builtin plugins
- my plugins
- builtin config (eg keymaps.lua)
- my config
- (lsp on_attach)

**Comment by amaanq** [comment]
in the order of your lazy setup

**Reply by hacker-DOM** [reply]
that answers the plugins section, but not the config section right?

**Reply by amaanq** [reply]
pretty sure its the same, your config should override lazyvims defaults
